crates/sui-upgrade-compatibility-transactional-tests/tests/snapshots/tests__struct_layout_change.snap

```diff
@@ -3,31 +3,31 @@ source: crates/sui-upgrade-compatibility-transactional-tests/tests/tests.rs
 expression: results
 ---
 ====
-Compatibility { check_struct_and_pub_function_linking: true, check_struct_layout: true, check_friend_linking: true, check_private_entry_linking: true }
+Compatibility { check_struct_and_pub_function_linking: true, check_struct_layout: true, check_friend_linking: true, check_private_entry_linking: true, disallowed_new_abilities: [] }
```

Comment:
Do you have context on these snapshot tests for compatibility? They are new to me and not sure how to interpret the change

Reply:
I do! (I added them 🙂) this is what I expect and totally fine  -- it's just printing out the compatibility setting for each one and since we added an extra field it's getting printed out here.